fecgov/fec-cms
# 3021 Upgrade the patch (also latest) version of lodash

## Summary

- Resolves #3021 

Upgrading the patch version of lodash will bring us to the latest version as well.

## Impacted areas of the application

Because it's only a patch version change, there should be no impact apart from addressing the security issues. Also any packages that use lodash automatically use the latest patch.

## Screenshots

None

## Related PRs

None

## How to test
- `Pull the branch`
- `npm i`
- Check that the site functions correctly, especially where lodash may be used
- Have Snyk scan the site again
____
